sdk/keyvault/azure-security-keyvault-secrets/CMakeLists.txt

```diff
@@ -20,23 +20,29 @@ include(AzureGlobalCompileOptions)
 az_vcpkg_integrate()
 
 if(NOT AZ_ALL_LIBRARIES)
-  find_package(azure-core-cpp "1.0.0" CONFIG QUIET)
+  find_package(azure-core-cpp "1.1.0" CONFIG QUIET)
   if(NOT azure-core-cpp_FOUND)
-    find_package(azure-core-cpp "1.0.0" REQUIRED)
+    find_package(azure-core-cpp "1.1.0" REQUIRED)
   endif()
 endif()
 
 set(
   AZURE_SECURITY_KEYVAULT_SECRETS_HEADER
     inc/azure/keyvault/secrets/dll_import_export.hpp
     inc/azure/keyvault/secrets/secret_client.hpp
+    inc/azure/keyvault/secrets/keyvault_secret.hpp
```

Comment:
This should probably be *keyvault_secrets.hpp* (plural) like the file below it. That also matches our package names across the other supported languages now.

Reply:
Secrets is the feature
in that header file we define a Secret, which is the object that is manipulated here. 
just like keys and key

Reply:
I see. That's the problem with GH PRs - difficult to see everything at a high level (I miss AzDO PRs).